index.html

```diff
@@ -1,299 +1,731 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Reem's Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-</head>
-<body>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
     <div class="container">
-       <div class="top-header">
-        <header >
-           <div class="nav-container">
+      <div class="top-header">
+        <header>
+          <div class="nav-container">
             <a>
-            <div id="logo">
-              Reem Muhammed
-            </div>
+              <div id="logo">Reem Muhammed</div>
```

Comment:
- [ ] Please ensure to remove the horizontal scrolling as it negatively impacts user experience.
**NB:** I found that adding `overflow-x: hidden;` to the `body` element in your CSS can get rid of this.
![image](https://user-images.githubusercontent.com/46839548/188148528-a15c41b2-57d7-4a2b-8c39-bbe3d1dfc2b5.png)